frontend/src/translations/ua/common.json

```
@@ -0,0 +1,566 @@
+{
+  "app": {
+    "name": "NEAR Explorer"
+  },
+  "common": {
+    "accounts": {
+      "account": "Акаунт",
+      "accounts": "Акаунти"
+    },
+    "blocks": {
+      "blocks": "Blocks",
+      "status": {
+        "fetching_status": "Отримання статусу...",
+        "checking_finality": "Перевірка остаточності...",
+        "finalized": "Завершено",
+        "finalizing": "Завершаємо"
+      }
+    },
+    "transactions": {
+      "transaction": "Транзакція",
+      "transactions": "Транзакції",
+      "status": {
+        "fetching_status": "Отримання статусу...",
+        "NotStarted": "не розпочато",
+        "Started": "розпочато",
+        "Failure": "Сталася помилка",
+        "SuccessValue": "успішно"
+      },
+      "execution": {
+        "convert_transaction_to_receipt": "Convert Transaction To Receipt",
+        "gas_burned": "Gas Burned",
+        "tokens_burned": "Tokens Burned"
```

```suggestion
        "convert_transaction_to_receipt": "Конвертація транзакції в квитанцію",
        "gas_burned": "Використано газу",
        "tokens_burned": "Використано токенів"
```
I'm not sure about this naming, @frol maybe you have any suggestions?

---

@shelegdmitriy Looks good to me 👍